docusaurus/docs/get-started/get-started.mdx

```diff
@@ -38,15 +38,15 @@ Scaffold a new plugin with a single command! Run the following and answer the pr
 
 ## Why create a Grafana plugin?
 
-Grafana plugin development allows you to create many different types of user experiences using [supported](/introduction/grafana-plugin-sdk-for-go) features. For example, you can make:
+Grafana plugin development allows you to create many different types of user experiences using [supported](https://grafana.com/developers/plugin-tools/key-concepts/grafana-plugin-sdk-for-go) features. For example, you can make:
```

Comment:
this link is broken. Could you also use a relative path?

Reply:
fixed